indico/core/signals/event/registration.py

```diff
@@ -18,8 +18,13 @@
 
 from blinker import Namespace
 
+
 _signals = Namespace()
 
+registration_personal_data_modified = _signals.signal('registration-personal-data-modified', """
+Called when the registration personal data is modified. The `sender` is the
+registration; the change is passed in the `change` kwarg.
```

Comment:
Shouldn't you say here "The `sender` is the `Registration` object"? (as you do right below)